test/evergreen.yml

```diff
@@ -1758,11 +1758,25 @@ buildvariants:
   run_on:
   - ubuntu1804-power8-test
   expansions:
-    configure_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH
+    configure_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH CFLAGS="-ggdb -fPIC"
```

Comment:
We should add CC and CXX flags as well.

Reply:
Updated the configure_env_vars to use mongodb toolchain binaries

Reply:
Agreed. @raviprakashgiri29 would you mind adding those missing env settings (PATH, CC, CXX) for `macos-1012` build variant, `configure-combinations` task to make the `configure_env_vars` setting consistent within evergreen.yml?

Reply:
Yes, I agree. I have updated configure_env_vars setting consistent across evergreen.yml.